### PR TITLE
Adds Pattern support to Kafka Streams Topology Dev UI

### PR DESCRIPTION
**Pull request by dcotfr**
Proposed fix for #29411

### CODE REVIEW

File: extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/devui/TopologyParserContext.java [hunk 1]
```diff
@@ -71,20 +81,15 @@ String toGraph() {
...
                 res.add(" subgraph cluster" + e.getKey() + " {");
                 res.add("  label=\"Sub-Topology: " + e.getKey() + "\"; color=\"#C8C879\"; bgcolor=\"#FFFFDE\";");
                 e.getValue().forEach(v -> res.add("  " + v + ';'));
                 res.add(" }");
             });
-            for (int i = 0; i < subGraphs.size(); i++) {
-                res.add(" subgraph cluster" + i + " {");
-                res.add("  label=\"Sub-Topology: " + i + "\"; color=\"#C8C879\"; bgcolor=\"#FFFFDE\";");
-                res.add(" }");
-            }
```
Comment (gsmet) [inline extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/devui/TopologyParserContext.java:87]: Why did you remove this section?

**Review by gsmet — COMMENTED** [review]
Thanks! I added a small question.

Probably something for @ozangunalp and @phillip-kruger to review.

**Review by phillip-kruger — APPROVED** [review]
LGTM. @ozangunalp please also have a look

**Comment by phillip-kruger** [comment]
Merging. @ozangunalp please let us know if there is any concerns (and we can roll forward)
